client/cli/go/cmds/heketi_storage.go

```diff
@@ -68,6 +70,11 @@ func init() {
 		3,
 		"\n\tOptional: Replica value for durability type 'replicate'."+
 			"\n\tDefault is 3")
+	setupHeketiStorageCommand.Flags().StringVar(&heketiStorageOptions,
```

Comment:
can we use `StringSliceVar` instead of `StringVar`?

Reply:
We could, but then it would work differently from the option on `volume create`. If we were doing it from the start I would have suggested that as well. In this case I think that consistency beats cleanliness.